packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -470,7 +470,7 @@ export interface IOS {
    */
   usesAppleSignIn?: boolean;
   /**
-   * A boolean indicating if the app uses Push Notifications Broadcast option for Push Notifications capability should be enabled by EAS CLI during capability syncing.
+   *  A boolean indicating if the app uses Push Notifications Broadcast option for Push Notifications capability. If true, EAS CLI will use the value during capability syncing. If EAS CLI is not used, this configuration will not have any effect unless another tool is used to operate on it, so enable the capability manually on the Apple Developer Portal in that case.
```

Comment:
this got kind of pulled into here due to reasons described in https://github.com/expo/universe/pull/17358